tests/phpUnit/Api/MediaLibraryApiTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Tests\Api;
+
+use App\Entity\MediaPackage;
+use App\Entity\MediaPackageCategory;
+use App\Entity\MediaPackageFile;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\EntityManager;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\File\File;
+
+/**
+ * @internal
+ * @coversNothing
+ */
+class MediaLibraryApiTest extends WebTestCase
+{
+  private EntityManager $entity_manager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp(): void
+  {
+    static::$kernel = static::createKernel();
+    static::$kernel->boot();
+    $this->entity_manager = static::$kernel->getContainer()
+      ->get('doctrine')
+      ->getManager()
+      ;
+
+    fopen('/tmp/phpUnitTest', 'w');
+    $file = new File('/tmp/phpUnitTest');
+
+    $new_media_package = new MediaPackage();
+    $new_media_package->setName('Looks');
+    $new_media_package->setNameUrl('looks');
+    $this->entity_manager->persist($new_media_package);
+    $this->entity_manager->flush();
+
+    $new_media_package_cat = new MediaPackageCategory();
+    $new_media_package_cat->setName('Pocket Family');
+    $new_media_package_cat->setPackage(new ArrayCollection([$new_media_package]));
+    $this->entity_manager->persist($new_media_package_cat);
+    $this->entity_manager->flush();
+
+    $this->addMediaFile('Panda', $file, $new_media_package_cat, 'pocketcode', 'Catrobat');
+    $this->addMediaFile('Cat', $file, $new_media_package_cat, 'luna', 'CatrobatLuna');
+    $this->addMediaFile('Dog', $file, $new_media_package_cat, 'pocketcode', 'Catrobat');
+    $this->addMediaFile('Rabbit', $file, $new_media_package_cat, 'luna', 'CatrobatLuna');
+    $this->addMediaFile('Bear', $file, $new_media_package_cat, 'pocketcode', 'Catrobat');
+    $this->addMediaFile('Snake', $file, $new_media_package_cat, 'luna', 'CatrobatLuna');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function tearDown(): void
+  {
+    parent::tearDown();
+    $this->entity_manager->close();
+  }
+
+  public function testMedia(): void
```

Comment:
it would be way more readable if the scenarios are split into multiple different test cases. Would the performance be 
noticeable worse due to the multiple executions of "setUp"?